homeassistant/components/image_processing/facebox.py

```diff
@@ -104,11 +129,22 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     if DATA_FACEBOX not in hass.data:
         hass.data[DATA_FACEBOX] = []
 
+    ip = config[CONF_IP_ADDRESS]
+    port = config[CONF_PORT]
+    username = None
+    if CONF_USERNAME in config:
+        username = config[CONF_USERNAME]
```

Comment:
```py
username = config.get(CONF_USERNAME)
```
This will default username to `None` if the option is missing from the config.

Reply:
Good to know..!